docs/docs/ProgrammingGuide/run-on-ec2.md

```diff
@@ -1,5 +1,13 @@
 ---
+## **AWS Marketplace AMI**
 
+The easiest way to get started with BigDL is the BigDL AMI in the AWS Marketplace. It includes Apache Spark, BigDL,
+Jupyter Notebooks, Python and more. Within minutes you can create a Jupyter Notebook, upload some data and begin
+experimenting with Deep Learning.
```

Comment:
why the line break here?
and change to "experimenting with Deep Learning using BigDL"

Reply:
Hi Jason, not sure what you mean by the line break...

Do you think we should append "Please note that it is highly recommended... " directly to this paragraph?
I break the first paragraph into 3 lines to make each line have the length around 100. In the generated doc, they will appear to be in the same paragraph.

I'll change it to "experimenting with Deep Learning using BigDL"

Reply:
There is a line break between "Apache Spark, BigDL," and "Jupyter Notebooks, Python and more".

Don't break the line manually; the build tool will handle that.

Reply:
I see. I think the line break will not be visible in the generated doc. But I'll update the doc to be consistent with the project style.